rspec-tools/rspec_tools/validation/description.py

```diff
@@ -16,8 +16,35 @@ def parse_names(path):
   section_names_path = read_file(path)
   return [s.replace('* ', '').strip() for s in section_names_path if s.strip()]
 
+def parse_security_standard_links(descr):
+  link_nodes = descr.find_all('a')
+  security_standards_links: Dict[str: List] = {}
```

Comment:
```suggestion
  security_standards_links: Dict[str, List] = {}
```
This is a typo I assume